src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(operation, ...numbers) {
+      operation.apply(this, numbers);
+
+      return this;
+    },
+    add(...numbers) {
+      for (const num of numbers) {
+        this.result += num;
+      }
```

Comment:
method add should work only with one number, so doesn't make sense to user cycle `for` here
```suggestion
    add(num) {
      this.result += num;
```